core/templates/dev/head/pages/collection_editor/CollectionEditorStateService.js

```diff
@@ -107,6 +101,7 @@ oppia.factory('CollectionEditorStateService', [
             alertsService.addWarning(
               error ||
               'There was an error when loading the collection rights.');
+            _isLoadingCollection = false;
```

Comment:
This is because `_isLoadingCollection` is is `true` only when the collection rights finishes loading, putting it here means that there is a "termination" point in case loading the rights object fails.